ci-scripts/JenkinsFile-OAI-Container-GitHub

```diff
@@ -295,12 +295,11 @@ pipeline {
                     } catch (Exception e) {
                       echo 'OK if not present'
                     }
-                    // In case of the full build, we need the host RHEL8 credentials.
-                    if (params.REGRESSION_TEST) {
-                      sh('mkdir -p tmp/ca tmp/entitlement')
-                      sh('cp /etc/pki/entitlement/*pem tmp/entitlement')
-                      sh('sudo cp /etc/rhsm/ca/redhat-uep.pem tmp/ca')
-                    }
+                    // Copying the host RHEL8 credentials all the time
+                    sh('mkdir -p ./etc-pki-entitlement ./rhsm-conf ./rhsm-ca')
+                    sh('cp /etc/pki/entitlement/*pem ./etc-pki-entitlement')
```

Comment:
NIT^2: `/etc-pki-entitlement` seems to encode the target folder `/etc/pki/entitlement` in `Dockerfile.rhel8`. Should this here be similar/consistent for the other folders? I.e.,
* `/rhsm-conf` -> `/etc-rhsm`
* `/rhsm-ca` -> `/etc-rhsm-ca`

Reply:
> lgtm, just some very minor NIT comments you can choose to ignore. I did not check the oai Jenkins run and will leave this to you.

@nstng  It passed.